examples/metrics/src/main/java/com/google/cloud/opentelemetry/example/metrics/MetricsExporterExample.java

```diff
@@ -0,0 +1,67 @@
+package com.google.cloud.opentelemetry.example.metrics;
+
+import static java.util.Collections.singleton;
+
+import com.google.cloud.opentelemetry.metric.MetricConfiguration;
+import com.google.cloud.opentelemetry.metric.MetricExporter;
+import io.opentelemetry.api.OpenTelemetry;
+import io.opentelemetry.api.metrics.LongUpDownCounter;
+import io.opentelemetry.api.metrics.Meter;
+import io.opentelemetry.sdk.OpenTelemetrySdk;
+import io.opentelemetry.sdk.metrics.export.IntervalMetricReader;
+import java.io.IOException;
+import java.time.Duration;
+import java.util.Random;
+
+public class MetricsExporterExample {
+  private static final Meter METER =
+      OpenTelemetry.getGlobalMeter("instrumentation-library-name", "semver:1.0.0");
+  private static io.opentelemetry.sdk.metrics.export.MetricExporter metricExporter;
+  private static IntervalMetricReader intervalMetricReader;
+
+  public MetricsExporterExample() {
+    try {
+      MetricConfiguration configuration =
+          MetricConfiguration.builder().setDeadline(Duration.ofMillis(30000)).build();
+      metricExporter = MetricExporter.createWithConfiguration(configuration);
+      intervalMetricReader =
```

Comment:
should this be moved into the metric exporter creation?

Reply:
In the trace example I've made all methods static and extracted into `setupTraceExporter`: https://github.com/GoogleCloudPlatform/opentelemetry-operations-java/blob/master/examples/trace/src/main/java/com/google/cloud/opentelemetry/example/trace/TraceExporterExample.java

I think it would be similar to follow the same structure for metrics?